database/nostr-lmdb/Cargo.toml

```diff
@@ -3,20 +3,22 @@ name = "nostr-lmdb"
 version = "0.42.0"
 edition = "2021"
 description = "LMDB storage backend for nostr apps"
-authors.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
+authors = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
 readme = "README.md"
-rust-version = "1.72.0"
+rust-version.workspace = true
```

Comment:
Thanks, I've checked the MSRV again and seems to match the 1.70.0.
I'll cherry-pick this little change in a separated commit and let you know when to rebase.